src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -52,6 +54,9 @@ public class ScriptHost : JobHost
         private ScriptSettingsManager _settingsManager;
         private bool _shutdownScheduled;
 
+        // Map from an extension name to a http handler. 
+        private IDictionary<string, IAsyncConverter<HttpRequestMessage, HttpResponseMessage>> _customHttpHandlers = new Dictionary<string, IAsyncConverter<HttpRequestMessage, HttpResponseMessage>>(StringComparer.OrdinalIgnoreCase);
```

Comment:
We shouldn't have any http stuff like this in ScriptHost. I think this should be moved to WebScriptHostManager, where we handle all the other http function stuff (see WebScriptHostManager.HandleRequest).